src/loki_alert_rules/hacluster_no_quorum.rule

```diff
@@ -0,0 +1,10 @@
+groups:
+  - name: haproxy_no_quorum
+    rules:
+      - alert: HaproxyNoQuorumError
+        expr: rate({%%juju_topology%%} |~ `resource management disabled due to lack of quorum` [5m]) > 0
```

Comment:
Do you have at hand the documentation where the available keywords are defined?

Reply:
Not that I know of, I decided to filter for this sentence after looking at some example logs in my testing environment:
```
2025-01-23T08:36:28.836331+00:00 juju-69734a-8 pacemaker-schedulerd[14017]:  warning: Fencing and resource management disabled due to lack of quorum
```